opal/corelib/basic_object.rb

```diff
@@ -34,6 +34,10 @@ def !
     false
   end
 
+  def !=(other)
+    self != other
```

Comment:
Haha, yeah it might be a good idea to make this more explicit and write `!(self == other)` instead. Otherwise this will break when the compiler is fixed.